source/js/main.js

```diff
@@ -186,6 +190,19 @@ let main = {
     });
   },
 
+  bindGAEventTrackers() {
+    document.querySelector(`#see-more-modular-page`).addEventListener(`click`, () => {
+      let label = ``;
+      let pageHeader = document.querySelectorAll(`.cms h1`)[0];
+
+      if (pageHeader) {
+        label = pageHeader.innerText + ` - footer cta`;
```

Comment:
With template strings you shouldn't need to use concatenation anymore, just:

```
`${pageHeader.innerText} - footer cta`
```

Reply:
that was straight up c/p from my previous code . will update 😂